lib/rules/web/ocm_console/osd_creation_page.xyaml

```diff
@@ -586,6 +590,52 @@ check_aws_compute_node_instances_type:
             xpath: //h4[text()='16 vCPU 32 GiB RAM']/../../div/div[text()='c5.4xlarge']
         - selector:
             xpath: //h4[text()='16 vCPU 32 GiB RAM']/../../div/div[text()='Compute optimized']
+check_aws_byoc_compute_node_instances_type:
+    elements:
+        - selector:
+            xpath: //h4[text()='48 vCPU 192 GiB RAM']/../../div/div[text()='m5.12xlarge']
+        - selector:
+            xpath: //h4[text()='48 vCPU 192 GiB RAM']/../../div/div[text()='General purpose']
+        - selector:
+            xpath: //h4[text()='64 vCPU 256 GiB RAM']/../../div/div[text()='m5.16xlarge']
+        - selector:
+            xpath: //h4[text()='64 vCPU 256 GiB RAM']/../../div/div[text()='General purpose']
+        - selector:
+            xpath: //h4[text()='96 vCPU 384 GiB RAM']/../../div/div[text()='m5.24xlarge']
+        - selector:
+            xpath: //h4[text()='96 vCPU 384 GiB RAM']/../../div/div[text()='General purpose']
+        - selector:
+            xpath: //h4[text()='32 vCPU 256 GiB RAM']/../../div/div[text()='r5.8xlarge']
+        - selector:
+            xpath: //h4[text()='32 vCPU 256 GiB RAM']/../../div/div[text()='Memory optimized']
+        - selector:
+            xpath: //h4[text()='48 vCPU 384 GiB RAM']/../../div/div[text()='r5.12xlarge']
+        - selector:
+            xpath: //h4[text()='48 vCPU 384 GiB RAM']/../../div/div[text()='Memory optimized']
+        - selector:
+            xpath: //h4[text()='64 vCPU 512 GiB RAM']/../../div/div[text()='r5.16xlarge']
+        - selector:
+            xpath: //h4[text()='64 vCPU 512 GiB RAM']/../../div/div[text()='Memory optimized']
+        - selector:
+            xpath: //h4[text()='96 vCPU 768 GiB RAM']/../../div/div[text()='r5.24xlarge']
+        - selector:
+            xpath: //h4[text()='96 vCPU 768 GiB RAM']/../../div/div[text()='Memory optimized']
+        - selector:
+            xpath: //h4[text()='36 vCPU 72 GiB RAM']/../../div/div[text()='c5.9xlarge']
+        - selector:
+            xpath: //h4[text()='36 vCPU 72 GiB RAM']/../../div/div[text()='Compute optimized']
+        - selector:
+            xpath: //h4[text()='48 vCPU 96 GiB RAM']/../../div/div[text()='c5.12xlarge']
+        - selector:
+            xpath: //h4[text()='48 vCPU 96 GiB RAM']/../../div/div[text()='Compute optimized']
+        - selector:
+            xpath: //h4[text()='72 vCPU 144 GiB RAM']/../../div/div[text()='c5.18xlarge']
+        - selector:
+            xpath: //h4[text()='72 vCPU 144 GiB RAM']/../../div/div[text()='Compute optimized']
+        - selector:
+            xpath: //h4[text()='96 vCPU 192 GiB RAM']/../../div/div[text()='c5.24xlarge']
+        - selector:
+            xpath: //h4[text()='96 vCPU 192 GiB RAM']/../../div/div[text()='Compute optimized']
```

Comment:
It should be better to define a loop in case, and sent as variables h than hard code them at here